tests/ui/or-patterns/missing-bindings.stderr

```diff
@@ -79,6 +79,14 @@ LL |     let (A(A(..) | B(a), _) | B(A(a, _) | B(a))) = Y;
    |            |
    |            pattern doesn't bind `a`
 
+error[E0408]: variable `c` is not bound in all patterns
```

Comment:
what happened here... I assume this is related to adding `c` as a known symbol???
Yeah, I renamed `c` to `builtin_syntax` and the error changed in nightly. Looks like it's sorting the symbols by their index? That's strange and bad, but luckily not your problem.

Reply:
yeah I see, I'll fix it

Reply:
#114454